resources/migrations/006-alter-bookmark-tags.edn

```diff
@@ -0,0 +1,4 @@
+{:up ["ALTER TABLE bookmarks DROP COLUMN tags;
```

Comment:
Instead of dropping the entire column, we could actually migrate existing data by adding the `text[]` column first, moving tags to this new column(using an `UPDATE`), and then deleting the old column. We can do a further `ALTER TABLE` to rename the new column if we want to keep the column name the same. This isn't very important now as there's no real data in any running Leaflike instance, but might be worth getting into the habit of doing.